romeo/src/state.rs

```diff
@@ -0,0 +1,218 @@
+use bdk::bitcoin::{
+    Address as BitcoinAddress, Block, Transaction as BitcoinTransaction, Txid as BitcoinTxId,
+};
+use blockstack_lib::burnchains::Txid as StacksTxId;
+use blockstack_lib::chainstate::stacks::StacksTransaction;
+use blockstack_lib::types::chainstate::StacksAddress;
+use tracing::debug;
+
+use crate::config::Config;
+use crate::event::Event;
+use crate::event::TransactionStatus;
+use crate::task::Task;
+
+/// The whole state of the application
+#[derive(Debug, Clone, Default, serde::Serialize, serde::Deserialize)]
```

Comment:
Afaik we don't. I can remove it.

Reply:
Although it will probably come in handy for tests so if you don't mind I prefer keeping it.

Reply:
> Although it will probably come in handy for tests so if you don't mind I prefer keeping it.

I'm not sure why do we need it for tests. It's not a blocker, but it would be don't do this until we need it. Cloning of complex objects (especially deep cloning) is not a good practice, IMHO.

Reply:
Sure, I do agree with you on that one so I'll remove it as a follow-up.

Reply:
Created a follow-up PR https://github.com/stacks-network/sbtc/pull/76